scripts/deployArchetypeBatch.ts

```diff
@@ -0,0 +1,33 @@
+import { ethers, run } from "hardhat";
+
+const sleep = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
+async function main() {
+//   const ArchetypeBatch = await ethers.getContractFactory("ArchetypeBatch");
```

Comment:
these are supposed to be uncommented, right?

Reply:
Yeah the commented portion is to deploy, the uncommented right now is how we could send a tx to the contract.